### ISSUE TITLE
list-group in a form in a panel

### ISSUE THREAD

**Issue by poulpo** [twbs/bootstrap#18525]
Hi,
Using bootstrap 3, I would like to put a list-group in a panel, but I need to have a form wrapping the list-group (and not the panel). This does not render the way I want.
Problem is visible here http://jsbin.com/wodirosajo/edit
I know the fix is not complete, and I am not even sure a proper fix can be in scope (because we can't know all the cases to discard in the selector). If it is not in scope, what could be the best "bootstrapy" way to fix this problem in my own less files.
Thanks.
Edit : fixed jsbin

**Comment by twbs-lmvtfy** [comment]
Hi @poulpo!

You appear to have posted a live example (http://jsbin.com/wodirosajo/edit), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 81, column 5 thru line 82, column 9: Consecutive hyphens did not terminate a comment. ```` -- ```` is not permitted inside a comment, but e.g. ```` - - ```` is.
* line 88, column 5 thru column 11: Element ```` style ```` not allowed as child of element ```` body ```` in this context. (Suppressing further errors from this subtree.)

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Reposting to trigger revalidation: http://jsbin.com/wodirosajo/edit

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://jsbin.com/wodirosajo/edit), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 93, column 3 thru column 9: Stray start tag ```` style ````.
* line 93, column 3 thru column 9: Element ```` style ```` not allowed as child of element ```` body ```` in this context. (Suppressing further errors from this subtree.)

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
@poulpo Would a solution involving using a `div` instead of a `ul` be acceptable?

**Comment by poulpo** [comment]
@cvrebert Yeah, I think so.

**Comment by cvrebert** [comment]
@poulpo Then I suggest http://jsbin.com/masexe/edit?html,output

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://jsbin.com/masexe/edit), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 79, column 3 thru column 9: Stray start tag ```` style ````.
* line 79, column 3 thru column 9: Element ```` style ```` not allowed as child of element ```` body ```` in this context. (Suppressing further errors from this subtree.)

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by poulpo** [comment]
@cvrebert Great ! My form is used elsewhere too (the HTML is generated from factorized blocks), so there will be a useless .list-group in the other cases, but the additional 'padding-left: 0;' will not break anything \o/
Thanks
